.github/workflows/publish-tool.yml

```diff
@@ -30,56 +18,64 @@ on:
 jobs:
   Publish:
     if: ${{ github.event_name == 'push' }}
-    runs-on: windows-latest
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
       - name: Setup .NET SDK
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: ${{ inputs.dotnet_sdk_version }}
+          
+      - name: Setup NuGet
+        uses: NuGet/setup-nuget@v1
 
       - name: Download package artifacts
         uses: actions/download-artifact@v2
         with:
-          name: packages
-          path: artifacts/Packages/
+          name: ${{inputs.REPOSITORY_NAME}}-packages
+          path: artifacts/packages/
+
 
+      # FOR DEVELOPMENT
+      # 
+      - name: Publish on test MyGet-feed
+        if: ${{ github.ref == 'refs/heads/future' }} # dev branche is published to myget moryx
+        env:
+          MORYX_NUGET_APIKEY: ${{secrets.MYGET_TOKEN}}
+          MORYX_PACKAGE_TARGET: 'https://www.myget.org/F/moryx-private-test/api/v2/package'
+          MORYX_PACKAGE_TARGET_V3: 'https://www.myget.org/F/moryx-private-test/api/v3/index.json'
+        run: dotnet nuget push "artifacts/packages/" --api-key $MORYX_NUGET_APIKEY --source $MORYX_PACKAGE_TARGET --skip-duplicate --symbol-api-key $MORYX_NUGET_APIKEY --symbol-source $MORYX_PACKAGE_TARGET_V3
+    
+
+      # Use if dev branch 
       - name: Publish on MyGet-CI
         if: ${{ github.ref == 'refs/heads/dev' }} # dev branche is published to myget moryx
-        shell: pwsh
         env:
-          MORYX_BUILDNUMBER: ${{inputs.MORYX_BUILDNUMBER}}
-          MORYX_OPTIMIZE_CODE: ${{inputs.MORYX_OPTIMIZE_CODE}}
-          MORYX_BUILD_CONFIG: ${{inputs.MORYX_BUILD_CONFIG}}
-          DOTNET_SKIP_FIRST_TIME_EXPERIENCE: ${{inputs.DOTNET_SKIP_FIRST_TIME_EXPERIENCE}}
           MORYX_NUGET_APIKEY: ${{secrets.MYGET_TOKEN}}
           MORYX_PACKAGE_TARGET: 'https://www.myget.org/F/moryx/api/v2/package'
```

Comment:
If you have the time at the end please change the package targets to be parameters on the action. Since the targets differ between private and public repos 😊